Dictionaries/en_US_names_etc.xml

```diff
@@ -578,6 +578,7 @@
   <name>Journee</name>
   <name>Jovani</name>
   <name>Jovanni</name>
+  <name>Joy</name>

```

Comment:
Not possible. See [PR #416 comment] (https://github.com/SubtitleEdit/subtitleedit/pull/416#issuecomment-69643379).